app/controllers/orders_controller.rb

```diff
@@ -5,11 +5,13 @@ class OrdersController < ApplicationController
   customer_tab  :all
 
   before_action :authenticate_user!
-  before_action :check_acting_as,          except: [:cart, :add, :choose_account, :show, :remove, :purchase, :update_or_purchase, :receipt, :update]
-  before_action :init_order,               except: [:cart, :index, :receipt]
-  before_action :protect_purchased_orders, except: [:cart, :receipt, :confirmed, :index]
+  before_action :check_acting_as,          except: [:cart, :cart_order_details_count, :add, :choose_account, :show, :remove, :purchase, :update_or_purchase, :receipt, :update]
+  before_action :init_order,               except: [:cart, :cart_order_details_count, :index, :receipt]
+  before_action :protect_purchased_orders, except: [:cart, :cart_order_details_count, :receipt, :confirmed, :index]
   before_action :load_statuses, only: [:show, :update, :purchase, :update_or_purchase]
 
+  skip_after_action :verify_same_origin_request, only: [:cart_order_details_count]
```

Comment:
This might be worth a code comment like "Using fetch instead of jQuery/XHR, and the only thing this action does it update the cart count"